src/Components/BabylonSandbox/BabylonSandbox.tsx

```diff
@@ -0,0 +1,259 @@
+import * as BABYLON from '@babylonjs/core/Legacy/legacy';
+import '@babylonjs/loaders';
+import * as GUI from '@babylonjs/gui';
+import React, { useState, useEffect, useRef } from 'react';
+import {
+    IBabylonSandboxProps,
+    IBabylonSandboxStyleProps,
+    IBabylonSandboxStyles
+} from './BabylonSandbox.types';
+import { getStyles } from './BabylonSandbox.styles';
+import { classNamesFunction, useTheme, styled } from '@fluentui/react';
+import { createGUID } from '../../Models/Services/Utils';
+import { getBoundingBox } from '../3DV/SceneView.Utils';
+
+const getClassNames = classNamesFunction<
+    IBabylonSandboxStyleProps,
+    IBabylonSandboxStyles
+>();
+
+/**
+ * Sets up the camera for scene.
+ */
+function setupCamera(
+    scene: BABYLON.Scene,
+    arcRotate: boolean,
+    center?: BABYLON.Vector3,
+    radius?: number
```

Comment:
lets fix linting warnings like this one by running `>npm run lint:check` and `>npm run lint` to fix those
```suggestion
    _radius?: number
```